Gemfile

```diff
@@ -54,7 +54,7 @@ end
 group :development do
   gem 'quiet_assets'
   gem 'better_errors'
-  # gem 'binding_of_caller'
+  gem 'binding_of_caller'
```

Comment:
Ok. Will take it out of the PR. @andremedeiros, where did we need this for again?